### PR TITLE
docs(options.md): Change default value of `esModule`

### PR DESCRIPTION
**Pull request by imcvampire**
Change default value from `false` to `true` follow new behavior in `>13.0.0`

### CODE REVIEW

File: docs/en/options.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ module.exports = {
 ### esModule
 
 - type: `boolean`
-- default: `false`
+- default: `tr`
```
Comment (francois2metz) [inline docs/en/options.md]: You should say `true`.

Comment (imcvampire) [inline docs/en/options.md]: Sorry. I fixed!

File: docs/en/options.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ module.exports = {
 ### esModule
 
 - type: `boolean`
-- default: `false`
+- default: `true`
```
Comment (kazupon) [inline docs/en/options.md]: `vue-loader` is released v12.x later.
We should be mentioned about the version like the below:

```
default: `false` (v13.0+, `true`)
```

File: docs/en/options.md [hunk 3]
```diff
@@ -116,6 +116,7 @@ module.exports = {
 
 - type: `boolean`
 - default: `true`
+- In v12.x or below, default value is `false`
```
Comment (kazupon) [inline docs/en/options.md]: why?
please update the `esModule`.

Comment (imcvampire) [inline docs/en/options.md]: I'm sorry :(. My mistake. I reverted this commit!

File: docs/en/options.md (hunk 2 above)
Comment (imcvampire) [inline docs/en/options.md]: I added the note about behaviour in v12.x or below.

**Pull request merged**